src/omt/bitvector_optimizer.h

```diff
@@ -0,0 +1,52 @@
+/*********************                                                        */
+/*! \file bitvector_optimizer.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Yancheng Ou
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ ** in the top-level source directory and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Optimizer for BitVector type
+ **/
+
+#ifndef CVC4__OMT__BITVECTOR_OPTIMIZER_H
+#define CVC4__OMT__BITVECTOR_OPTIMIZER_H
+
+#include "omt_optimizer.h"
+
+namespace CVC4::smt {
+
+/**
+ * Optimizer for BitVector type
+ */
+class OMTOptimizerBitVector : public OMTOptimizer
+{
+ public:
+  OMTOptimizerBitVector(bool isSigned);
+  virtual ~OMTOptimizerBitVector() = default;
+  virtual std::pair<OptResult, Node> minimize(SmtEngine* parentSMTSolver,
```

Comment:
```suggestion
  std::pair<OptResult, Node> minimize(SmtEngine* parentSMTSolver,
```

Reply:
AFAIK Removing virtual will cause the compiler directly call OMTOptimizer::minimize when invoking [unique_ptr<OMTOptimizer>] ->minimize.

Reply:
I might be wrong, perhaps override has the same effect as virtual...

Reply:
OK I am wrong... specifying virtual in the base class is sufficient...

Reply:
Resolved